data/meson.build

```diff
@@ -142,7 +142,7 @@ if build_daemon
     )
   endif
 
-  if launchctl.found()
+  if get_option('launchd').enabled() and launchctl.found()
```

Comment:
Is enabled() right? I would think allowed()

Reply:
Fixed.